dali/base/dasds.cpp

```diff
@@ -1814,6 +1814,32 @@ struct CStoreInfo
     unsigned xmlCrc{0};
     unsigned binaryCrc{0};
     StringAttr cache;
+
+    void serialize(IFileIO *fileIO, unsigned *crcXml, unsigned *crcBinary)
+    {
+        assertex(fileIO);
+
+        // Only serialize xmlCrc and binaryCrc
+        if (crcXml)
+            xmlCrc = *crcXml;
```

Comment:
see comment where calls. This changes the semantics, it was not altering storeInfo members before.